frontend/src/component/layout/MainLayout/NavigationSidebar/NewInUnleash/NewInUnleashTooltip.tsx

```diff
@@ -179,6 +179,6 @@ export const NewInUnleashTooltip: FC<{
             </ClickAwayListener>
         }
     >
-        {children}
+        <div>{children}</div>
```

Comment:
To make events propagate correctly, you do, in fact, require one real HTML element to handle those clicks. This is a new problem that I created by moving the code into NewInUnleashSideBarItem

Reply:
Not a big deal I think